notebooks/bo_sphere.ipynb

```diff
@@ -194,23 +219,31 @@
     "## Definition of the BO surrogate model\n",
     "Here we use a Gaussian process with a Matérn kernel on $\\mathbb{S}^2$, as by [1].\n",
     "\n",
-    "The kernel has three positive real hyperparameters: vairance $\\sigma^2$, length scale $\\kappa$ and smoothness $\\nu$.\n",
-    "Definiting a Gaussian process model also requires a liklihood noise hyperparameter $\\sigma_n^2$.\n",
+    "The kernel has three positive real hyperparameters: variance $\\sigma^2$, length scale $\\kappa$ and smoothness $\\nu$.\n",
+    "Defining a Gaussian process model also requires a likelihood noise hyperparameter $\\sigma_n^2$.\n",
     "\n",
-    "We use the $\\operatorname{Gamma}(2.0, 0.15)$ prior for $\\sigma$, the squre root of $\\sigma^2$, and $\\operatorname{Gamma}(1.1, 0.05)$ prior for $\\sigma_n^2$."
+    "We use the $\\operatorname{Gamma}(2.0, 0.15)$ prior for $\\sigma$, the square root of $\\sigma^2$, and $\\operatorname{Gamma}(1.1, 0.05)$ prior for $\\sigma_n^2$."
```

Comment:
Lol, the density of typos here is impressive. My mind must have been partly elsewhere when I wrote this :)